packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -906,6 +906,25 @@ export default class Meeting extends StatelessWebexPlugin {
     this.hasJoinedOnce = false;
   }
 
+  /**
+   * Fetches meeting information for public use.
+   * @public
+   * @memberof Meeting
+   * @returns {Promise}
+   */
+  fetchMeetingInfoForMeeting() {
```

Comment:
There is already one on the webex.meetings object , I don't think we need one here also

Reply:
@arun3528 the one on webex.meetings object would not update the meeting info in the correct meeting instance, so Cantina needs to call fetchMeetingInfo on the meeting object and initially when Natalia started working on this the existing fetchMeetingInfo() was taking destination and type as arguments, so she needed a version that doesn't need these args.
However, now after recent Colin's changes the existing `meeting.fetchMeetingInfo()` no longer takes destination and type as arguments, so @nataliadelmar we don't need this new method anymore, you can just move the logic from it into the existing `meeting.fetchMeetingInfo()` method